tokio/src/runtime/task/mod.rs

```diff
@@ -361,6 +361,13 @@ impl<S: 'static> Task<S> {
     fn header_ptr(&self) -> NonNull<Header> {
         self.raw.header_ptr()
     }
+
+    cfg_taskdump! {
+        pub(super) fn notify_for_tracing(self) -> Notified<S> {
```

Comment:
You should be able to fix that like this:
```suggestion
        pub(super) fn notify_for_tracing(&self) -> Notified<S> {
```

Reply:
That's a compile error. `Notified` wraps a `Task`, not a `&Task`.
```text
error[E0308]: mismatched types
   --> tokio/src/runtime/task/mod.rs:368:22
    |
368 |             Notified(self)
    |             -------- ^^^^ expected `Task<S>`, found `&Task<S>`
    |             |
    |             arguments to this struct are incorrect
    |
    = note: expected struct `runtime::task::Task<S>`
            found reference `&runtime::task::Task<S>`
```

Reply:
Since you created a new refcount, you can create a new `Task` using the `raw` field.

Reply:
Got it, thanks! Done.